compiler/ocaml.vim

```diff
@@ -31,19 +17,46 @@ let current_compiler = "ocaml"
 let s:cpo_save = &cpo
 set cpo&vim
 
-CompilerSet errorformat =
-      \%EFile\ \"%f\"\\,\ lines\ %*\\d-%l\\,\ characters\ %c-%*\\d:,
-      \%EFile\ \"%f\"\\,\ line\ %l\\,\ characters\ %c-%*\\d:,
-      \%EFile\ \"%f\"\\,\ line\ %l\\,\ characters\ %c-%*\\d\ %.%#,
+" The OCaml compiler comes in many flavours, so pick up a sensible default
+" here as far as the compiler plugin is concerned
+CompilerSet makeprg=ocamlc\ %
```

Comment:
This is not working for me.
The default for `makeprg` is `make`, which is widely used, at least as documentation.
I'm used to fix my `makeprg` but this time I was confused to see completely unexpected type errors and it took me a lot of time to find that my makeprg was messed up.

This interferes with my own configuration that sets `dune` as the default. Overriding the default must be possible and documented as this will not work in most OCaml project and people will not be happy with this.